scripts/benchmarks_ci.py

```diff
@@ -237,13 +237,22 @@ def __main(args: list) -> int:
     # Run micro-benchmarks
     if not args.build_only:
         for framework in args.frameworks:
+            #Before we run the benchmarks we need to set the CommitDate in the environment
+            #for the nex reporting tool
+            target_framework_moniker = micro_benchmarks.FrameworkAction.get_target_framework_moniker(framework)
+            commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli)
+            source_timestamp = dotnet.get_commit_date(framework, commit_sha, args.cli_repository)
+            os.environ['PERFLAB_HASH'] = commit_sha
+            os.environ['PERFLAB_BUILDTIMESTAMP'] = source_timestamp
             micro_benchmarks.run(
                 BENCHMARKS_CSPROJ,
                 args.configuration,
                 framework,
                 verbose,
                 args
             )
+            os.environ['PERFLAB_HASH'] = ""
```

Comment:
I would just remove them from the environment instead of setting them to empty string.

Reply:
maybe use something like `os.unsetenv` or `os.environ.pop`?